build.xml

```diff
@@ -26,7 +26,7 @@
 			optimize="true"
 			debug="true"
 			source="1.6" 
-			target="1.6">
+			target="1.7">
```

Comment:
1468 will still be on Java 1.6, so this shouldn't be updated quite yet. I can cherry-pick and avoid this one, so no need to do anything.